README.md

```diff
@@ -47,8 +47,6 @@ case classes remain.
 - [IntelliJ IDEA](https://www.jetbrains.com/help/idea/discover-intellij-idea-for-scala.html):
   the most widely used IDE for Scala using a re-implementation of the Scala
   typechecker.
-- [Scala IDE](http://scala-ide.org/): Eclipse-based IDE using the Scala
```

Comment:
I also just went ahead and removed this. I really don't think it's a recommended alternative anymore. I saw a tweet this weekend about how mentioning this in various places and tutorial often cause confusion for newcomers to Scala, and I thought it was a valid point. I think we should remove this.